### PR TITLE
Rename Compute {Element -> Entrypoint} (SOFTWARE-4265)

### PR DESCRIPTION
**Pull request by edquist**

### CODE REVIEW

File: docs/compute-entrypoint/htcondor-ce-overview.md [hunk 1]
```diff
@@ -13,10 +13,10 @@ Before continuing with the overview, make sure that you are familiar with the fo
 -   Pilot jobs, frontends, and factories (i.e., [GlideinWMS](http://glideinwms.fnal.gov/doc.prd/index.html),
     AutoPyFactory)
 
-What is a Compute Element?
+What is a Compute Entrypoint?
 --------------------------
 
-An OSG Compute Element (CE) is the entry point for the OSG to your local resources: a layer of software that you install
+An OSG Compute Entrypoint (CE) is the entry point for the OSG to your local resources: a layer of software that you install
 on a machine that can submit jobs into your local batch system.
```
Comment (brianhlin) [inline docs/compute-entrypoint/htcondor-ce-overview.md:20]:
```suggestion
An OSG Compute Entrypoint (CE) is the door for remote organizations to submit requests to temporarily allocate local
compute resources.
```

I didn't love the doubling up of "Entrypoint" and "entry point"

**Review by brianhlin — CHANGES_REQUESTED** [review]
Let's not mess with the file locations until we update MkDocs, which should give us redirects so that we don't leave folks with old links hanging.

File: docs/compute-entrypoint/htcondor-ce-overview.md (hunk 1 above)
Comment (edquist) [reply docs/compute-entrypoint/htcondor-ce-overview.md:20]: Knock on the OSG Door of Opportunity (DO) !

File: docs/detailed-overview.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Add OSG Software
 If necessary, provision all OSG hosts that are in your site plan that do not exist yet.
 The general steps to installing an OSG site are:
 
-1. Install [OSG Yum Repos](/common/yum) and the [Compute Element software](#installing-and-configuring-the-compute-element)
+1. Install [OSG Yum Repos](/common/yum) and the [Compute Entrypoint software](#installing-and-configuring-the-compute-entrypoint)
```
Comment (brianhlin) [inline docs/detailed-overview.md:44]:
```suggestion
1. Install [OSG Yum Repos](/common/yum) and the [Compute Entrypoint software](#installing-and-configuring-the-compute-element)
```

**Review by brianhlin — APPROVED** [review]
One last link but otherwise LGTM

**Review by brianhlin — APPROVED** [review]
LGTM

**Pull request merged**